imap_processing/tests/swe/test_swe_l1a_science.py

```diff
@@ -29,22 +29,20 @@ def test_swe_raw_science_data(decom_test_data):
         index_col="SHCOARSE",
     )
 
-    first_data = decom_test_data[0]
-    validation_data = raw_validation_data.loc[first_data.data["SHCOARSE"].raw_value]
+    first_data = decom_test_data.isel(epoch=0)
+    validation_data = raw_validation_data.loc[first_data["shcoarse"].values]
 
-    # compare raw values of housekeeping data
-    for key, value in first_data.data.items():
-        if key == "SHCOARSE":
-            # compare SHCOARSE value
-            assert value.raw_value == validation_data.name
-            continue
-        if key == "SCIENCE_DATA":
-            continue
-        # check if the data is the same
-        assert value.raw_value == validation_data[key]
+    # compare raw values of the packets
+    shared_keys = set([x.lower() for x in validation_data.keys()]).intersection(
+        first_data.keys()
+    )
+    # TODO: Why are all the fields not the same between the two
```

Comment:
Can you clarify this?

Reply:
I'm curious why all the validation data isn't the same as the dataset we are producing. I would think that our dataset should contain all the values that are in the validation data, but there is a difference between the validation data keys, and the created dataset keys.

These are the variables that are in one or the other, but not both.
```
{'cem1', 'phverno', 'cem3', 'phapid', 'phgroupf', 'cem2', 'phtype', 'phseqcnt', 'timestamp', 'cem5', 'cem4', 'cem6', 'cem7', 'phdlen', 'phshf'}
```

Reply:
OK, would you like me to remove the comment or update it to something else?

Reply:
If you can add comment to validate those CEM# data when we get better data, that would be great.

Reply:
:+1: Sounds good. I just updated the docstring.